components/popover/index.js

```diff
@@ -258,6 +278,9 @@ class Popover extends Component {
 			className,
 			'is-' + yAxis,
 			'is-' + xAxis,
+			{
+				'is-mobile': this.state.isMobile,
```

Comment:
Does this need to be state? Can't we just call `isMobile()`?

Reply:
yes, because we need to rerender if it changes.